src/components/JoinGroup/JoinGroupForm.tsx

```diff
@@ -13,36 +13,41 @@ import { useSelector } from 'react-redux'
 const Button = styled(MuiButton)(spacing)
 
 export interface JoinGroupValues {
-    id: string
-    password: string
-    username: string
+    groupId: string
+    groupPassword: string
+    displayName: string
 }
 
 const JoinGroupForm = ({ onFormSubmit }) => {
     const { t } = useTranslation('join-group-form')
     const token = useSelector(selectToken)
 
+    const validateForm = (values) => {
```

Comment:
still not yup, thus needing manual validatio which is not helping reducing dev time

Reply:
I copy/pasted from `AdminDashboard`, I didn't study yup yet